phys2bids/tests/test_txt.py

```diff
@@ -1,185 +1,127 @@
+    txt.populate_phys_input(samefreq_short_txt_file, chtrig=2)
+    # testing for LabChart files
+    txt.populate_phys_input(multifreq_lab_file, chtrig=1)
+
+
+@pytest.mark.parametrize('units, expected', [
+    ('0.001 s', 1000),
+    ('0.001 min', 16.666666666666668),
+    ('0.001 hr', 0.2777777777777778),
+    ('1 ms', 1000),
+    ('1000 µs', 1000)
+])
+def test_process_labchart(loaded_lab_file, units, expected):
+    header, channels, chtrig = loaded_lab_file
+    header[0][1] = units
+    phys_obj = txt.process_labchart(channels, chtrig=chtrig, header=header)
+    assert math.isclose(phys_obj.freq[0], expected)
+
+
+def test_process_labchart_notime(notime_lab_file):
     chtrig = 0
-    phys_obj = txt.process_labchart(channels_no_time, chtrig, header_no_time)
-    assert len(phys_obj.timeseries) == len(channels_no_time[0]) + 1
+    header, channels = txt.read_header_and_channels(notime_lab_file, chtrig)
+    phys_obj = txt.process_labchart(channels, chtrig=chtrig, header=header)
+    assert len(phys_obj.timeseries) == len(channels[0]) + 1
 
 
-def test_process_acq(test_read_header_and_channels):
-    header = test_read_header_and_channels[0]
-    channels = test_read_header_and_channels[1]
-    chtrig = 2
+def test_process_labchart_errors(loaded_lab_file):
+    header, channels, chtrig = loaded_lab_file
+
     # test file without header
     with raises(AttributeError) as errorinfo:
-        txt.process_acq(channels, chtrig)
+        txt.process_labchart(channels, chtrig=chtrig)
     assert 'not supported' in str(errorinfo.value)
-    # test when units are msec:
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 10000
-    # test when units are msec:
-    header[1][0] = '0.01 sec/sample'
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 100
-    # test when units are in µsec:
-    header[1][0] = '1 µsec/sample'
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 1000000.0
-    # test when units are in Hz:
-    header[1][0] = '100 Hz'
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 100
-    # test when units are in kHz:
-    header[1][0] = '1 kHz'
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 1000
-    # test when units are in MHz:
-    header[1][0] = '1 MHz'
-    phys_obj = txt.process_acq(channels, chtrig, header)
-    assert phys_obj.freq[0] == 1000000
+
+    # test when units are not valid for process_labchart
+    header[0][1] = ' 1 gHz'
+    with raises(AttributeError) as errorinfo:
+        txt.process_labchart(channels, chtrig=chtrig, header=header)
+    assert 'not in a valid LabChart' in str(errorinfo.value)
+
+
+@pytest.mark.parametrize('units, expected', [
+    ('1 msec/sample', 1000),
+    ('0.01 sec/sample', 100),
+    ('1 µsec/sample', 1000000.0),
+    ('100 Hz', 100),
+    ('1 kHz', 1000),
+    ('1 MHz', 1000000)
+])
```

Comment:
Could we add a test for ``min`` units? It's not being tested according to the codecov report and is one of the units accepted in the acq format

Reply:
For sure! Just something like `'0.001 min/sample'`? The expected number should be `16.6666666` or something like that, I believe, in that case. (I'll test it out to make sure.)

Let me know if you'd prefer something else.

Reply:
perfect